Treewide: remove inactive maintainer viric

AndersonTorres opened the pull request:
## Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

https://github.com/NixOS/nixpkgs/issues/290642

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

AndersonTorres committed:
release-lib: remove viric from crossMaintainers [orphans]

Since theey is not active from at least six years.

AndersonTorres committed:
treewide: remove viric from meta.maintainers [no orphans]

Since theey is not active from at least six years.

AndersonTorres commented:
For the future merger:
Can we relax the restriction here and merge it ignoring the formatter?

Mic92 reviewed (approved):
Fine with me let's wait 1-2 days for a response.

viric reviewed (approved):
I confirm I'm an inactive maintainer.

Mic92 commented:
Okay. Than we can merge it right away.

tobim commented:
I want to add myself for `perf`, `ghostscript`, and `logrotate`. Can you make that change or should I open a follow up PR?

AndersonTorres commented:
> I want to add myself for `perf`, `ghostscript`, and `logrotate`. Can you make that change or should I open a follow up PR?

Better:
I will remove them from this PR. This way you can adopt them right now.

Aleksanaa commented:
As the first person to encounter the unused with error due to this, can you share your thoughts on this? 😄🎙️

Mic92 commented on pkgs/applications/audio/lingot/default.nix:
```diff
@@ -46,6 +46,6 @@ stdenv.mkDerivation rec {
     license = lib.licenses.gpl2Plus;
     platforms = with lib.platforms; linux;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 replied:
Mhm. I was hoping, I could just commit these straight to the branch, it it seems like github no longer allows me to add those.

AndersonTorres replied:
This is a bad problem from GitHub: it does not allow edits from upstream maintainers when the PR comes from an organization.

Mic92 commented on pkgs/applications/graphics/meshlab/default.nix:
```diff
@@ -95,7 +95,7 @@ mkDerivation rec {
     homepage = "https://www.meshlab.net/";
     license = lib.licenses.gpl3Only;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/applications/graphics/wings/default.nix:
```diff
@@ -48,7 +48,7 @@ stdenv.mkDerivation rec {
     description = "Subdivision modeler inspired by Nendo and Mirai from Izware";
     license = lib.licenses.tcltk;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/applications/misc/librecad/default.nix:
```diff
@@ -72,7 +72,7 @@ mkDerivation rec {
     homepage = "https://librecad.org";
     license = licenses.gpl2Only;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/applications/networking/instant-messengers/silc-client/default.nix:
```diff
@@ -33,7 +33,7 @@ stdenv.mkDerivation rec {
     mainProgram = "silc";
     license = lib.licenses.gpl2;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/applications/networking/irc/irssi/fish/default.nix:
```diff
@@ -22,7 +22,7 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/falsovsky/FiSH-irssi";
     license = licenses.mit;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/applications/networking/offrss/default.nix:
```diff
@@ -36,7 +36,7 @@ stdenv.mkDerivation rec {
     description = "Offline RSS/Atom reader";
     license = licenses.agpl3Plus;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/applications/networking/siproxd/default.nix:
```diff
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
     description = "Masquerading SIP Proxy Server";
     mainProgram = "siproxd";
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/applications/office/mmex/default.nix:
```diff
@@ -71,7 +71,7 @@ stdenv.mkDerivation rec {
     homepage = "https://www.moneymanagerex.org/";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/applications/science/electronics/caneda/default.nix:
```diff
@@ -19,7 +19,7 @@ mkDerivation rec {
     homepage = "http://caneda.org";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/applications/science/electronics/xoscope/default.nix:
```diff
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
     homepage = "https://xoscope.sourceforge.net";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/applications/science/math/yacas/default.nix:
```diff
@@ -80,7 +80,7 @@ stdenv.mkDerivation rec {
     homepage = "http://www.yacas.org/";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/applications/search/doodle/default.nix:
```diff
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
     description = "Tool to quickly index and search documents on a computer";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/by-name/zs/zsync/package.nix:
```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
     description = "File distribution system using the rsync algorithm";
     license = licenses.artistic2;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/development/libraries/flann/default.nix:
```diff
@@ -73,7 +73,7 @@ stdenv.mkDerivation rec {
     license = lib.licenses.bsd3;
     description = "Fast approximate nearest neighbor searches in high dimensional spaces";
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/development/libraries/vxl/default.nix:
```diff
@@ -44,7 +44,7 @@ stdenv.mkDerivation (finalAttrs: {
     # (and see https://github.com/InsightSoftwareConsortium/ITK/pull/1920/files for potential patch)
     license = [ lib.licenses.unfree ];
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/development/python-modules/aiohttp-apispec/default.nix:
```diff
@@ -45,6 +45,6 @@ buildPythonPackage rec {
     homepage = "https://github.com/maximdanilchenko/aiohttp-apispec/";
     license = licenses.mit;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/development/tools/misc/cbrowser/default.nix:
```diff
@@ -30,7 +30,7 @@ stdenv.mkDerivation rec {
     homepage = "https://sourceforge.net/projects/cbrowser/";
 
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/development/tools/misc/cscope/default.nix:
```diff
@@ -47,7 +47,7 @@ stdenv.mkDerivation rec {
     homepage = "https://cscope.sourceforge.net/";
 
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/development/tools/profiling/malt/default.nix:
```diff
@@ -28,7 +28,7 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/memtt/malt";
     license = licenses.cecill-c;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/games/bsdgames/default.nix:
```diff
@@ -63,7 +63,7 @@ stdenv.mkDerivation rec {
     description = "Ports of all the games from NetBSD-current that are free";
     license = lib.licenses.free;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/games/gltron/default.nix:
```diff
@@ -25,7 +25,7 @@ stdenv.mkDerivation rec {
     mainProgram = "gltron";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/games/neverball/default.nix:
```diff
@@ -45,7 +45,7 @@ stdenv.mkDerivation rec {
     description = "Tilt the floor to roll a ball";
     license = "GPL";
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/games/pioneers/default.nix:
```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
     homepage = "https://pio.sourceforge.net/";  # https does not work
     license = licenses.gpl2Plus;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/games/torcs/default.nix:
```diff
@@ -49,7 +49,7 @@ stdenv.mkDerivation rec {
     homepage = "https://torcs.sourceforge.net/";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/games/trigger/default.nix:
```diff
@@ -65,7 +65,7 @@ stdenv.mkDerivation rec {
     homepage = "http://trigger-rally.sourceforge.net/";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/games/ufoai/default.nix:
```diff
@@ -43,7 +43,7 @@ stdenv.mkDerivation rec {
     description = "Squad-based tactical strategy game in the tradition of X-Com";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/games/ultimatestunts/default.nix:
```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation rec {
     description = "Remake of the popular racing DOS-game Stunts";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/games/vdrift/default.nix:
```diff
@@ -52,7 +52,7 @@ let
       homepage = "http://vdrift.net/";
       license = lib.licenses.gpl2Plus;
-      maintainers = with lib.maintainers; [ viric ];
+      maintainers = with lib.maintainers; [ ];
```
```suggestion
      maintainers = [ ];
```

Mic92 commented on pkgs/os-specific/linux/netatop/default.nix:
```diff
@@ -51,6 +51,6 @@ stdenv.mkDerivation {
     license = lib.licenses.gpl2Only;
     platforms = lib.platforms.linux;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/servers/silc-server/default.nix:
```diff
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
     mainProgram = "silcd";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/X11/xbindkeys/default.nix:
```diff
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
     description = "Launch shell commands with your keyboard or your mouse under X Window";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [ viric ];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/X11/xdotool/default.nix:
```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
     description = "Fake keyboard/mouse input, window management, and more";
     license = lib.licenses.bsd3;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/X11/xtrace/default.nix:
```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
     description = "Tool to trace X11 protocol connections";
     license = licenses.gpl2Only;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/admin/tigervnc/default.nix:
```diff
@@ -153,7 +153,7 @@ stdenv.mkDerivation rec {
     license = lib.licenses.gpl2Plus;
     description = "Fork of tightVNC, made in cooperation with VirtualGL";
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/backup/btar/default.nix:
```diff
@@ -32,6 +32,6 @@ stdenv.mkDerivation rec {
     homepage = "https://viric.name/cgi-bin/btar";
     platforms = platforms.all;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/compression/pbzip2/default.nix:
```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
     description = "Parallel implementation of bzip2 for multi-core machines";
     license = licenses.bsd2;
-    maintainers = with maintainers; [viric];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/filesystems/mtdutils/default.nix:
```diff
@@ -39,7 +39,7 @@ stdenv.mkDerivation rec {
     license = licenses.gpl2Plus;
     homepage = "http://www.linux-mtd.infradead.org/";
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/graphics/exiftags/default.nix:
```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
     description = "Displays EXIF data from JPEG files";
     license = lib.licenses.free;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/misc/pal/default.nix:
```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
     description = "Command-line calendar program that can keep track of events";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/misc/xburst-tools/default.nix:
```diff
@@ -43,7 +43,7 @@ stdenv.mkDerivation {
     license = lib.licenses.gpl3;
     homepage = "http://www.linux-mtd.infradead.org/";
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/networking/filegive/default.nix:
```diff
@@ -18,7 +18,7 @@ buildGoModule rec {
     description = "Easy p2p file sending program";
     license = licenses.agpl3Plus;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/networking/mosh/default.nix:
```diff
@@ -59,7 +59,7 @@ stdenv.mkDerivation rec {
     '';
     license = licenses.gpl3Plus;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/networking/netkit/tftp/default.nix:
```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
     homepage = "ftp://ftp.uk.linux.org/pub/linux/Networking/netkit/";
     license = lib.licenses.bsdOriginal;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/networking/olsrd/default.nix:
```diff
@@ -30,7 +30,7 @@ stdenv.mkDerivation rec {
     license = lib.licenses.bsd3;
     homepage = "http://olsr.org/";
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/networking/pdnsd/default.nix:
```diff
@@ -32,6 +32,6 @@ stdenv.mkDerivation rec {
     license = licenses.gpl3Plus;
     platforms = platforms.unix;
-    maintainers = with maintainers; [viric];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/networking/pwnat/default.nix:
```diff
@@ -34,7 +34,7 @@ stdenv.mkDerivation rec {
     description = "ICMP NAT to NAT client-server communication";
     license     = lib.licenses.gpl3Plus;
-    maintainers = with maintainers; [viric];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/networking/udptunnel/default.nix:
```diff
@@ -19,7 +19,7 @@ stdenv.mkDerivation rec {
     description = "Tunnels TCP over UDP packets";
     license = lib.licenses.gpl3Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/security/ccrypt/default.nix:
```diff
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
     description = "Utility for encrypting and decrypting files and streams with AES-256";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/system/fakeroot/default.nix:
```diff
@@ -79,7 +79,7 @@ stdenv.mkDerivation (finalAttrs: {
     description = "Give a fake root environment through LD_PRELOAD";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/system/gt5/default.nix:
```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
     homepage = "https://gt5.sourceforge.net/";
     license = lib.licenses.gpl2Plus;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/system/idle3tools/default.nix:
```diff
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
     description = "Tool to get/set the infamous idle3 timer in WD HDDs";
     license = lib.licenses.gpl3;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/system/tm/default.nix:
```diff
@@ -22,7 +22,7 @@ stdenv.mkDerivation rec {
     homepage = "http://vicerveza.homeunix.net/~viric/soft/tm";
     license = licenses.gpl2Plus;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/system/ts/default.nix:
```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
     description = "Task spooler - batch queue";
     license = licenses.gpl2Only;
-    maintainers = with maintainers; [ viric ];
+    maintainers = with maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/text/mairix/default.nix:
```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
     description = "Program for indexing and searching email messages stored in maildir, MH or mbox";
     mainProgram = "mairix";
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented on pkgs/tools/text/multitran/mtutils/default.nix:
```diff
@@ -29,7 +29,7 @@ stdenv.mkDerivation rec {
     mainProgram = "mtquery";
     license = lib.licenses.gpl2Only;
-    maintainers = with lib.maintainers; [viric];
+    maintainers = with lib.maintainers; [ ];
```
```suggestion
    maintainers = [ ];
```

Mic92 commented:
I will fix empty maintainer lists using the `with` expression in a follow up PR.

AndersonTorres committed:
treewide: remove viric from meta.maintainers [orphans]

Since theey is not active from at least six years.

All the packages on this commit became orphans.

---------------------------------------------------------------------------
There are files not covered by this commit, because they will be adopted
soon. Namely:

- pkgs/by-name/zs/zsync/package.nix
- pkgs/games/bsdgames/default.nix
- pkgs/misc/ghostscript/default.nix
- pkgs/os-specific/linux/kernel/perf/default.nix
- pkgs/tools/system/logrotate/default.nix

AndersonTorres commented:
> As the first person to encounter the unused with error due to this, can you share your thoughts on this? 😄🎙️

I have mixed feelings about this.

In a first sight, I left the packages that way because a not so long time ago it was recommended - something about "it would be easier to add new maintainers in the future".

On the other hand, now that CI reports this as an error, there is no more room to never-ending discussions.

The net result of being shot by an error was positive!

Mic92 commented:
https://github.com/NixOS/nixpkgs/pull/330664

Mic92 commented:
I think overall having detection for unused with statements is quite useful, while having an exception just for `maintainer` fields would make it only more complex.

matthiasbeyer commented:
The discussion moved away from the actual topic of the PR AFAICS... how about we merge this?

AndersonTorres commented:
> I will fix empty maintainer lists using the `with` expression in a follow up PR.

I can solve this with a bit of emacs-fu. Now CI is happy!
Nonetheless the cleanup will be needed in other expressions...

Mic92 commented:
I thought, I pressed the merge button already... must have timed out.

bendlas commented:
This removal PR has already been merged, despite not being grounded within community consensus. See https://github.com/NixOS/nixpkgs/pull/337478#issuecomment-2310779183

@viric  be sure that we value your involvement and any time you may have to spare, helping with the packages you've maintained and others as well, and if you decide to come back, please don't hesitate to be added to maintainers again.
